XLS-20/README.md

```diff
@@ -824,4 +829,34 @@ This functionality is intended to allow the `owner` of an **`NFToken`** to offer
 :memo: If both offers are for the same asset, it is possible that the order in which funds are transferred might cause a transaction that would succeed to fail due to an apparent lack of funds. To ensure deterministic transaction execution and maximimize the chances of successful execution, this proposal requires that the account attempting to buy the **`NFToken`** is debited first and that funds due to the broker are credited _before_ crediting the seller or issuer.
 
 :note: In brokered mode, The offers referenced by `NFTokenBuyOffer` and `NFTokenSellOffer` must both specify the same `NFTokenID`; that is, both must be for the same **`NFToken`**.
-      
\ No newline at end of file
+      
+## 1.6. Uniqueness property of the `NFTokenID`
+The `NFTokenID` is ensured to be unique by using its `Sequence` number structure, and by imposing a restriction on deleting accounts.
+### 1.6.1. NFT `Sequence` Construct
+The `Sequence` of a NFT is the lowest 32-bit of its `NFTokenID`. It is computed by adding the `FirstNFTokenSequence` with `MintedNFTokens` to produce a monotonically increasing number. 
+
+By adding the `FirstNFTokenSequence` offset, we can prevent the NFT `Sequence` from starting at 0 whenever the issuer recreates their account. This helps to ensure that the `NFTokenID` remains unique. 
```

Comment:
"Adding the `FirstNFTokenSequence` offset prevents...."

(Avoid using first person singular.)